app/src/main/java/com/infomaniak/mail/utils/SentryDebug.kt

```diff
@@ -227,18 +227,16 @@ object SentryDebug {
             "Message is in wrong Folder (related to 'Message has multiple parent folders')",
             SentryLevel.ERROR,
         ) { scope ->
-            scope.setExtra(
-                "localMessageFolders",
-                "${localMessage?.foldersForSentry?.joinToString { "${it.role?.name} | ${it.id}" }}",
-            )
+            scope.setExtra("localMessage folderId", "${localMessage?.folderId}")
+            scope.setExtra("remoteMessage folderId", remoteMessage.folderId)
```

Comment:
Won't this be bothersome to follow in Sentry? Some of these captureMessages will have one type of extras and other will have another type of extras